tests/acceptance/courses-test.js

```diff
@@ -24,9 +24,36 @@ test('visiting /courses', async function(assert) {
   assert.equal(currentPath(), 'courses');
 });
 
+test('visiting /courses with title filter', async function(assert) {
+  server.create('academicYear', {id: 2014});
+  server.create('course', {
+    title: 'specialfirstcourse',
+    year: 2014,
+    school: 1,
+  });
+  server.create('course', {
+    title: 'specialsecondcourse',
+    year: 2014,
+    school: 1
+  });
+  server.create('course', {
+    title: 'regularcourse',
+    year: 2014,
+    school: 1
+  });
+  let lastCourse = server.create('course', {
+    title: 'aaLastcourse',
+    year: 2014,
+    school: 1
+  });
+  await visit('/courses?filter=Last');
+  assert.equal(find('.courses-list [data-test-courses] tr').length, 1);
```

Comment:
Should be
assert.equal(page.courses().count, 1);
(even if `page.visit` doesn't work this will.